apps/package-manager/packages/generic/src/coreHandler.ts

```diff
@@ -253,10 +257,11 @@ export class CoreHandler {
 			}
 
 			if (this.deviceSettings['delayRemoval'] !== this.delayRemoval) {
-				this.delayRemoval = this.deviceSettings['delayRemoval']
+				this.delayRemoval = Number(this.deviceSettings['delayRemoval']) || DEFAULT_DELAY_REMOVAL_PACKAGE
```

Comment:
If the delayRemoval is set to "0" (ie user wants no removal delayl),  this will be set to the default value

Reply:
Which was kind-of on purpose, because I don't think a retention of 0 makes any production sense. But I guess this can be "any non-negative value"?